boards/arm/mec15xxevb_assy6853/pinmux.c

```diff
@@ -44,7 +44,7 @@ static int board_pinmux_init(struct device *dev)
 	 * controlled by their respective PCR register (UART2).
 	 * For more details see table 44-1
 	 */
-	ECS_REGS->DEBUG_CTRL |= 0x4;
+	ECS_REGS->DEBUG_CTRL = 0x5;
```

Comment:
Use MCHP MCHP_ECS_DCTRL_DBG_EN  and others to be clear in the intention

Reply:
done